api/test/integration/env/configurations/vulnerability/manager/healthcheck/healthcheck.py

```diff
@@ -5,7 +5,7 @@
 
 sys.path.append('/tools')
 
-from healthcheck_utils import base_url, check, get_login_header, get_manager_health_base, get_response, login_url
+from healthcheck_utils import BASE_URL, check, get_login_header, get_manager_health_base, get_response, LOGIN_URL
```

Comment:
About the constants, does this mean that it was broken? I don't see changes on any other module and you are importing a "different" object.